src/error.rs

```diff
@@ -18,6 +18,9 @@ pub enum FromEnvErrorKind {
     /// There is no jobserver in the environment variable.
     /// Variables associated with Make can be used for passing data other than jobserver info.
     NoJobserver,
+    /// Environment variable contains negative file descriptors which means
+    /// jobserver is disabled for this process
```

Comment:
What about something like:

```suggestion
    /// The jobserver style is simple pipe, but at least one of the file descriptors
    /// is negative, which means it is disabled for this process.
    ///
    /// See ... (the link that is now in the function docs)
```

Reply:
Thanks! Done (a bit changed, hopefully it won't be a problem)